drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/data_warehouse_api.rb

```diff
@@ -0,0 +1,100 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisExternalApis::AcHmis
+  class DataWarehouseApi
+    SYSTEM_ID = 'ac_hmis_warehouse'.freeze
```

Comment:
For the "MCI ID," the MCI ID ExternalIDs namespace and the MCI RemoteCredential slug (where they're fetched from) are the same (`ac_hmis_mci`). Here, they're different (slug `ac_hmis_warehouse`, and namespace `ac_hmis_mci_unique_id`). I think that's fine, just pointing it out. We don't (and shouldn't) expect them to be the same.

However I think this SYSTEM_ID should maybe be more descriptive. Later on we will need to add a RemoteCredential for dropping an export file to an sftp server ([ticket](https://www.pivotaltracker.com/story/show/185190925)) for the same warehouse, so that remote credential might have a slug like something like `ac_hmis_warehouse_sftp`. Perhaps this one should be `ac_hmis_warehouse_api`?

Reply:
I noticed this difference as well, and made them different purposefully. I'll update this one like you propose.